### PR TITLE
Add operations for credential storage related to web3id.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Add functionality for interacting with credential storage contract, storing and retrieving.

## Checklist

- [ ] My code follows the style of this project.
- [ ] The code compiles without warnings.
- [ ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src/web3id.rs [hunk 1]
```diff
@@ -181,3 +180,172 @@ pub async fn get_public_data(
...
+
+    /// The parameter type for the contract function `serializationHelper`.
+    #[derive(concordium_std::Serialize, serde::Serialize, serde::Deserialize, Clone, Debug)]
+    pub struct DataToSign {
+        /// A timestamp to make signatures expire.
+        pub timestamp:            Timestamp,
+        /// The contract_address that the signature is intended for.
+        pub contract_address:     ContractAddress,
+        /// Metadata associated with the credential.
+        pub version:              u16,
```
Comment (annenkov) [inline src/web3id.rs:287]: This field is called `metadata` in the credential storage contract, but it's called "version" here. Maybe it should be metadata everywhere for consistency, as the documentation above says.

**Review by annenkov — APPROVED** [review]
Looks good to me.

File: src/web3id.rs (hunk 1 above)
Comment (abizjak) [reply src/web3id.rs:287]: Version is what it is, so I want to keep it like this in the SDK API.

Maybe you want to make a PR changing the name in the contract?

Comment (abizjak) [reply src/web3id.rs:287]: I have updated the documentation.

**Pull request merged**